.travis.yml

```diff
@@ -26,17 +26,16 @@ matrix:
     - node_js: 8
       env: COVERAGE=true
 
-
 install:
-  - npm run bootstrap
+  - yarn bootstrap
 
 cache:
   directories:
     - node_modules
     - $(npm config get prefix)/lib/node_modules # globally installed stuff (i.e. lerna)
```

Comment:
```suggestion
    - $(yarn config get prefix)/lib/node_modules # globally installed stuff (i.e. lerna)
```

Reply:
Hm I just tried this locally and it doesn't seem to work. Have you tried it?

Reply:
Hmm, seems to work on my end?

```
➜  truffle git:(quorum-subprovider) ✗ $(yarn config get prefix)/lib/node_modules
➜  node_modules ls
yarn
➜  node_modules 
```

Reply:
What happens if you run `yarn config get prefix` by itself? I get `undefined`

Reply:
same, which is why we pass the expected path where the global `node_modules` would be installed/cached.

Reply:
This seems wonky with how it returns undefined. We'll just have to keep an eye to make sure the Travis cache behaves.